fluent-dom/src/dom_localization.js

```diff
@@ -132,6 +129,19 @@ export default class DOMLocalization extends Localization {
       }
     }
 
+    if (this.windowElement) {
+      if (this.windowElement !== newRoot.ownerGlobal) {
+        throw new Error(`Cannot connect a root:
```

Comment:
Grammar nit: `a root` doesn't sound right here; I don't know how to make it right. `Cannot connect root` maybe?

Reply:
Verified with bz - `a root` is correct.